deploy/helm-chart/templates/deployment.yaml

```
@@ -154,4 +160,7 @@ spec:
         emptyDir: {}
       - name: nginx-lib
         emptyDir: {}
+      {{- with .Values.extraVolumes }}
```

to get rid of the newline after the existing volumes:
```suggestion
      {{- with .Values.extraVolumes -}}
```